src/main/java/oshi/hardware/stores/HWNetworkStore.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) 2016 com.github.dblock.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * enrico[dot]bianchi[at]gmail[dot]com
+ *    com.github.dblock - initial API and implementation and/or initial documentation
+ */
+package oshi.hardware.stores;
+
+import javax.json.Json;
+import javax.json.JsonBuilderFactory;
+import javax.json.JsonObject;
+import oshi.json.NullAwareJsonObjectBuilder;
+import oshi.json.OshiJsonObject;
+
+/**
+ * Store object of network interfaces attributes
+ *
+ * @author enrico[dot]bianchi[at]gmail[dot]com
+ */
+public class HWNetworkStore implements OshiJsonObject {
```

Comment:
I'm not sure the "store" is applicable to networks... was ok for FileStores (software) and DiskStores (hardware).  Probably want `NetworkInterface`